### PR TITLE
Inline anatomy SVGs

### PR DESCRIPTION
**Pull request by arnoldsandoval-okta**

### CODE REVIEW

**Review by edburyenegren-okta — APPROVED** [review]
This rules. SVG-friendly theming, here we come!

File: packages/vuepress-theme-odyssey/global-components/Anatomy.vue [hunk 1]
```diff
@@ -1,6 +1,14 @@
 <template>
-  <figure v-if="img" class="docskit-figure-anatomy">
-    <img :src="img" :alt="alt" />
+  <figure v-if="img" 
+      :class="{
+      'docskit-figure-anatomy': true,
+      [`is-figure-anatomy-filled`]: fill
+    }"
```
Comment (jmelberg-okta) [inline packages/vuepress-theme-odyssey/global-components/Anatomy.vue:6]: style-nit: Spacing

```suggestion
      :class="{
        'docskit-figure-anatomy': true,
        [`is-figure-anatomy-filled`]: fill
      }"
```

File: packages/vuepress-theme-odyssey/global-components/Anatomy.vue [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+  >
+    <div
+      v-html="require(`!html-loader!../../docs/.vuepress/public${img}`)"
```
Comment (jmelberg-okta) [inline packages/vuepress-theme-odyssey/global-components/Anatomy.vue:9]: Suggest separating `${img}` and `public` with a slash and update `img` tags accordingly, as it'll be much easier to read.

File: packages/docs/.vuepress/public/images/anatomy-modal.svg [hunk 3]
```diff
@@ -1,5 +1,4 @@
 <svg width="958" height="455" viewBox="0 0 958 455" fill="none" xmlns="http://www.w3.org/2000/svg">
-<rect width="958" height="455" fill="#EBEBED"/>
```
Comment (jmelberg-okta) [inline packages/docs/.vuepress/public/images/anatomy-modal.svg:2]: Did this get removed by running SVGO? Curious why this is only now a problem.

**Review by jmelberg-okta — APPROVED** [review]
Nothing blocking - few nits.

File: packages/docs/.vuepress/public/images/anatomy-modal.svg (hunk 3 above)
Comment (arnoldsandoval-okta) [reply packages/docs/.vuepress/public/images/anatomy-modal.svg:2]: Joe had baked in a background to the image, so I removed it. By inlining the SVG in a div, it created gaps in the Visual component. I added a fill prop to that component which provides the background color for us.

File: packages/vuepress-theme-odyssey/global-components/Anatomy.vue (hunk 1 above)
Comment (arnoldsandoval-okta) [reply packages/vuepress-theme-odyssey/global-components/Anatomy.vue:6]: Thanks for catching this, I was relying on eslint to catch this sort of stuff. Ill take another look at the config.

**Pull request merged**